sdk/resourcemanager/azure-resourcemanager-appservice/src/main/java/com/azure/resourcemanager/appservice/implementation/KuduClient.java

```diff
@@ -326,17 +334,20 @@ private InputStreamFlux fluxFromInputStream(InputStream inputStream) {
                 flux ->
                     flux
                         .zipWith(
-                            Flux.range(1, 30),
+                            Flux.range(1, 6),
                             (Throwable throwable, Integer integer) -> {
                                 if (throwable instanceof ManagementException
                                         && ((ManagementException) throwable).getResponse().getStatusCode() == 502
-                                    || throwable instanceof JsonParseException) {
+                                    || throwable instanceof JsonParseException
+//                                    || throwable instanceof TimeoutException
+//                                    || throwable instanceof SocketTimeoutException
```

Comment:
why add two commented statement?

Reply:
Timeout exception from netty/okhttp respectively. But I cannot reproduce it in track2, so just keep them commented.